packages/devtools_app/lib/src/shared/analytics/constants/_inspector_constants.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of '../constants.dart';
+
+/// Events for opening documentation for the inspector tab.
+enum InspectorDocs {
+  packageDirectories('Package Directories');
+
+  const InspectorDocs(this.name);
+  final String name;
```

Comment:
no need to add the name field. It is built into the dart enum. Change this to `packageDirectoriesDocs`, that way the value that comes through analytics will be 'packageDirectoriesDocs' and not 'packageDirectories' which isn't as clear.